src/components/thumbnail/collapsible-document-section.tsx

```diff
@@ -41,7 +41,7 @@ export const CollapsibleDocumentsSection: React.FC<IProps> = observer(
       // get the personal documents
       networkResource.teachers?.forEach((teacher) => {
         if (teacher.personalDocuments) {
-          for (const [key, document] of Object.entries(teacher.personalDocuments)) {
+          for (const [key] of Object.entries(teacher.personalDocuments)) {
             documentKeys.push(key);
           }
```

Comment:
If you're only going to be using the keys, you should be able to do something like
```suggestion
          documentKeys.push(...Object.keys(teacher.personalDocuments));
```
here and in several other places below.